osx/post_installation.sh

```diff
@@ -0,0 +1,169 @@
+        exit 1
+    fi
+}
+
+function create_plist {
+    # Create Plist 
+    echo "Now creating '$PLIST_SRC'..."
+    echo "<?xml version='1.0' encoding='UTF-8'?>" >> $PLIST_SRC
+    echo "<!DOCTYPE plist PUBLIC '-//Apple//DTD PLIST 1.0//EN' 'http://www.apple.com/DTDs/PropertyList-1.0.dtd'>" >> $PLIST_SRC
+    echo "<plist version='1.0'>" >> $PLIST_SRC
+    echo "<dict>" >> $PLIST_SRC
+    echo -e "\t<key>Label</key>" >> $PLIST_SRC
+    echo -e "\t<string>org.learningequality.kalite</string>" >> $PLIST_SRC
+    echo -e "\t<key>ProgramArguments</key>" >> $PLIST_SRC
+    echo -e "\t<array>" >> $PLIST_SRC
+    echo -e "\t\t<string>sh</string>" >> $PLIST_SRC
+    echo -e "\t\t<string>-c</string>" >> $PLIST_SRC
+    echo -e "\t\t<string>launchctl setenv KALITE_PYTHON \"$PYRUN\"</string>" >> $PLIST_SRC
+    echo -e "\t</array>" >> $PLIST_SRC
+    echo -e "\t<key>RunAtLoad</key>" >> $PLIST_SRC
+    echo -e "\t<true/>" >> $PLIST_SRC
+    echo "</dict>" >> $PLIST_SRC
+    echo "</plist>" >> $PLIST_SRC
+}
+
+#----------------------------------------------------------------------
+# Script
+#----------------------------------------------------------------------
+
+ENV=$(env)
+syslog -s -l error "Packages post-installation initialize with env:'\n'$ENV" 
+
+STEP=1
+STEPS=9
+
+echo "Now preparing KA-Lite dependencies..."
+
+echo "$STEP/$STEPS. Symlink kalite executable to /usr/local/bin..."
+if [ ! -d "$SYMLINK_TO" ]; then
+    echo ".. Now creating '$SYMLINK_TO'..."
+    sudo mkdir -p $SYMLINK_TO
+    if [ $? -ne 0 ]; then
+        echo ".. Abort!  Error encountered creating '$SYMLINK_TO' directory."
+        exit 1
+    fi
+fi
+
+
+if [ -f "$KALITE" ]; then
+    echo ".. Found $KALITE executable, it will be removed and will create new one."
+    rm -fr $KALITE
+fi
+
+
+$COMMAND_SYMLINK
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Error encountered running '$COMMAND_SYMLINK'."
+    exit 1
+fi
+
+((STEP++))
+echo "$STEP/$STEPS. Export KALITE_PYTHON env that point to Pyrun directory..."
+update_env
+
+((STEP++))
+echo "$STEP/$STEPS. Create plist in ~/Library/LaunchAgents folders..."
+if [ -f "$PLIST_SRC" ]; then
+    echo ".. Found an existing '$PLIST_SRC', now removing it."
+    rm -fr $PLIST_SRC
+fi
+
+
+if [ ! -d "$LAUNCH_AGENTS" ]; then
+    echo ".. Now creating '$LAUNCH_AGENTS'..."
+    sudo mkdir -p $LAUNCH_AGENTS
+    if [ $? -ne 0 ]; then
+        echo ".. Abort!  Error encountered creating '$LAUNCH_AGENTS' directory."
+        exit 1
+    fi
+fi
+
+create_plist
+
+((STEP++))
+echo "$STEP/$STEPS. check the BIN_PATH that points to the python/pyrun interpreter to use..."
+$PYRUN $SHEBANGCHECK_PATH/shebangcheck.py
+if [ $? -ne 0 ]; then
+    echo ".. Abort!  Error encountered running '$SHEBANGCHECK_PATH/shebangcheck.py'."
+    exit 1
+fi
+
+((STEP++))
+echo "$STEP/$STEPS. Running kalite manage syncdb --noinput..."
+$BIN_PATH/kalite manage syncdb --noinput
+
+((STEP++))
+echo "$STEP/$STEPS. Running kalite manage init_content_items --overwrite..."
```

Comment:
@djallado Pls put a "REF: https://github.com/learningequality/ka-lite/issues/4682#issuecomment-159113225" for reference here.  Also a "TODO" to remove this when it's fixed later.